### PR TITLE
Add time and memory to XML report

### PR DESCRIPTION
**Pull request by keradus**
Continuation of #1369

@junichi11 this PR is based on your commits.
Do you want to add change you have mentioned [here](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1369#issuecomment-137007123) ?

cc @SpacePossum

### CODE REVIEW

File: Symfony/CS/Console/Command/FixCommand.php [hunk 1]
```diff
@@ -449,6 +449,39 @@ protected function execute(InputInterface $input, OutputInterface $output)
...
                 }
 
+                $fixEvent = $this->stopwatch->getEvent('fixFiles');
+
+                $timeXML = $dom->createElement('time');
+                $memoryXML = $dom->createElement('memory');
+                $filesXML->appendChild($timeXML);
+                $filesXML->appendChild($memoryXML);
+
+                $memoryXML->setAttribute('value', round($fixEvent->getMemory() / 1024 / 1024, 3));
```
Comment (GrahamCampbell) [inline Symfony/CS/Console/Command/FixCommand.php]: shame that code is duplicated

**Comment by SpacePossum** [comment]
Looks good to me, valid xml :+1:  
About the format itself, we can both ways and I have no strong feelings about one or the other.
Just pick something that we hope most people like since we cannot change it until 3.x (but you probably are all aware of that) ;)

**Comment by GrahamCampbell** [comment]
What? We can change anything we want in 2.x. It's not released yet. Nothing needs to wait until 3.x?

**Comment by junichi11** [comment]
@keradus 
> Do you want to add change you have mentioned here ?

Yes, I think that we should change it. When new tags will be added later, they can be added easily to the root tag(`report` or `phpcsfixer`?). It is easier to understand that the `report` tag is the root tag than the `files` tag.

**Comment by SpacePossum** [comment]
:+1: for changing the root node from `files` to `report` :)

**Comment by keradus** [comment]
Thank you @junichi11.

**Comment by keradus** [comment]
@junichi11, could you please send a second PR that create new root ?

**Comment by junichi11** [comment]
@keradus Sure. I'll do that :)

**Pull request closed without merging**